config/config-types.js

```diff
@@ -24,9 +24,9 @@ export type NetworkConfigType = {
   backendUrl: string,
   websocketUrl: string,
   name: Network,
-  trezorNetwork:
-  1 // testnet
-  | 2, // Mainnet
+  trezorProtocolMagic:
+    764824073   // mainnet
+  | 1097911063, // testnet
```

Comment:
I would change the wording from `testnet` to `staging` (there is a testnet chain and it uses a diff protocolMagic)

Reply:
we should also change the protocolMagic to `633343913`